### DISCUSSION TITLE
AnimatedLayout exiting element with delay makes touchables not trigger anymore.

### DISCUSSION THREAD

**Discussion by Sixzero** [software-mansion/react-native-reanimated#2186] [Q&A]
So the title seems to be true as for now, and would like to know what alternatives we have.

Could we by any chance still use touch events on the exiting element?

What I would expect is that, the exiting and entering elements are just the same elements as  they were before transitions.

An MVP Task component could be for testing:
```
const Task = ({ }) => {
  return (
    <Animated.View
      entering={SlideInRight}
      exiting={SlideOutRight.delay(4000)} // This is the important delay for testing.
      layout={Layout.springify()}
      style={{ ... }}
    >
      <TouchableOpacity
        onPress={() => {
          console.log('WE WERE INSIDE:');
        }}
      >
        <View style={{ height: 40, width: 40 }}/>
      </TouchableOpacity>
    </Animated.View>
  );
};
```

**Comment by terrysahaidak** [comment]
Could you please try the Touchable from Gesture Handler?
